content/docs/1.17/monitoring.md

```diff
@@ -41,3 +41,7 @@ Jaeger has the ability to trace some of its own components, namely the requests
 ```
 docker run -e JAEGER_DISABLED=true -p 16686:16686 jaegertracing/all-in-one:{{< currentVersion >}}
 ```
+
+## Prometheus monitoring mixin for Jaeger
```

Comment:
Q: the dashboard covers metrics only, correct? Maybe it's better to move this into a subsection under Metrics above?

Reply:
Sure thing, done.